hat/assets/js/cypress/integration/02 - forms/details.spec.js

```diff
@@ -26,28 +29,139 @@ const formVersionsPageOne = makePaginatedResponse({
 });
 
+let interceptFlag = false;
+
+const updateInterceptFlag = value => {
+    interceptFlag = value;
+};
+
 describe('Forms details', () => {
     beforeEach(() => {
         cy.login();
         cy.intercept('GET', '/sockjs-node/**');
         cy.intercept('GET', '/api/profiles/me/**', superUser);
-        cy.intercept('GET', '/api/v2/orgunittypes/**', orgUnitTypes);
-        cy.intercept('GET', '/api/projects/**', projects);
-        // TODO paramatrise form_id
+        cy.intercept('GET', '/api/v2/orgunittypes/**', orgUnitTypes).as(
+            'orgUnitTypes',
+        );
+        cy.intercept('GET', '/api/projects/**', projects).as('projects');
+        // TODO parametrise form_id
         cy.intercept(
             'GET',
             '/api/formversions/?&limit=20&order=-version_id&form_id=1',
             formVersionsPageOne,
-        );
+        ).as('getFormVersions');
         cy.intercept(
             'GET',
             // eslint-disable-next-line max-len
-            '/api/forms/1/?fields=id,name,org_unit_types,projects,period_type,derived,single_per_period,periods_before_allowed,periods_after_allowed,device_field,location_field,label_keys,possible_fields',
+            '/api/forms/1/?fields=id,name,org_unit_types,projects,period_type,derived,single_per_period,periods_before_allowed,periods_after_allowed,device_field,location_field,label_keys,possible_fields,legend_threshold,change_request_mode',
             form,
-        );
+        ).as('getForm');
         cy.visit(baseUrl);
+        cy.wait('@orgUnitTypes');
+        cy.wait('@projects');
+    });
+    describe('Filters', () => {
+        it('renders only base filters by default', () => {
+            cy.getAndAssert('#input-text-name');
+            cy.getAndAssert('#period_type');
+            cy.getAndAssert('#project_ids');
+            cy.getAndAssert('#org_unit_type_ids');
+            cy.get('#input-text-device_field').should('not.exist');
+            cy.get('#input-text-location_field').should('not.exist');
+            cy.get('#label_keys').should('not.exist');
+            cy.get('#change_request_mode').should('not.exist');
+            cy.get('#input-text-periods_before_allowed').should('not.exist');
+            cy.get('#input-text-periods_after_allowed').should('not.exist');
+            cy.get('[data-test="single_per_period"]').should('not.exist');
+        });
+    });
+    describe('When mounting', () => {
+        testPermission(baseUrl);
+        testTopBar(baseUrl, 'Form', true);
     });
-    describe.skip('Filters', () => {});
+    describe('When updating', () => {
+        it('Save button is disabled if no project', () => {
+            cy.wait(['@getForm', '@getFormVersions']);
+            // cy.get('#project_ids').as('projectsInput').click();
+            // cy.get('@projectsInput')
+            //     .parent()
+            //     .within(() => {
+            //         cy.get('[data-testid="ClearIcon"]').click();
+            //     });
```

Comment:
🗑️  ?

Reply:
Yes, the dropdown is selection is already empty, so trying to find the clear button will cause the test to fail